StRoot/StETofMatchMaker/StETofMatchMaker.cxx

```diff
@@ -295,6 +305,14 @@ StETofMatchMaker::InitRun( Int_t runnumber )
         return kStFatal;
     }
 
+    
+    // --------------------------------------------------------------------------------------------
+    // pointer to eTOF hit maker
+    // --------------------------------------------------------------------------------------------
+
+    mETofHitMaker = ( StETofHitMaker* ) GetMaker( "etofHit" );
+    LOG_DEBUG << "StETofMatchMaker::InitRun() -- pointer to eTOF hit maker: " << mETofHitMaker << endm;
+    
```

Comment:
No maker should directly access another maker. This would entangle the dependency between makers.